src/test/groovy/tests/PrometheusTest.groovy

```diff
@@ -204,7 +176,7 @@ class PrometheusTest extends DataAnalyticsSteps{
     void rtAuditByUsersTest() throws Exception {
         def query = "sum by (user) (increase(jfrog_rt_access_audit{user!=\"UNKNOWN\", user!=\"_system_\",user!=\"\"}[10m]))"
```

Comment:
the `""` double quotes in groovy creates a `GString` (not related to underwear). You would use it for interpolation in your strings. However, since you're not doing any interpolation here, if you change it to `''` single quotes (classic java string), then you can remove the quote escapes `\"` - it would make it more readable